Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -174,7 +174,7 @@ instance instLinearOrderedCommMonoidWithZeroMultiplicativeOrderDual
     LinearOrderedCommMonoidWithZero (Multiplicative αᵒᵈ) where
   zero := Multiplicative.ofAdd (OrderDual.toDual ⊤)
   zero_mul := @top_add _ (_)
-  -- Porting note:  Here and elsewhere in the file, just `zero_mul` worked in Lean 3. See
+  -- Porting note:  Here and elsewhere in the file, just `top_add` worked in Lean 3. See
```

Comment:
I think the porting note was probably correct? Why the change?

If it's just that we expect never to use allow that because of stricter type synonyms, I think we should just delete the note.